src/Console/Command/FixCommand.php

```diff
@@ -184,7 +184,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $progressOutput = $showProgress && $stdErr
-            ? new ProcessOutput($stdErr, $this->eventDispatcher)
+            ? new ProcessOutput($stdErr, $this->eventDispatcher, count($resolver->getFinder()))
```

Comment:
On the project (445 files), it takes ~0.05s (basic `microtime()` test). Is it really that expensive?

I tried to store the files in an array and use it for both counting and running the fixers. Don't know if it reduces execution time but it consumes 4M memory.